templates/actors/vehicle/vehicle-sheet.html

```diff
@@ -162,7 +162,7 @@ <h4 class="attribute-label">{{localize "Cost"}} (GC)</h4>
         <div class="attribute-box">
           <div class="attribute-section">
             <div class="attribute-name">
-              <h4 class="attribute-label">{{localize "Length"}} (ft.)</h4>
+              <h4 class="attribute-label">{{localize "Length"}} ({{localize "ft"}}.)</h4>
```

Comment:
I don't see a key for this and should probably have a more specific one than `ft`